### DISCUSSION TITLE
Level Organization

### DISCUSSION THREAD

**Discussion by zicklag** [fishfolk/jumpy#797] [General]
@erlend-sh I was thinking that maybe we should make Level 9 the first level, because it's a great beginner introduction, and it works much better with the AI, which are inevitably effecting the first-impression of the game.

We might want to re-order all of the maps by how well they are working. Anyway, figured I'd open the discussion to collect thoughts.

**Comment by erlend-sh** [comment]
Yeah great idea 👍
